eventlogger/pom.xml

```diff
@@ -100,6 +100,16 @@
             <groupId>com.fasterxml.jackson.datatype</groupId>
             <artifactId>jackson-datatype-jsr310</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.slack.api</groupId>
+            <artifactId>slack-api-client</artifactId>
+            <version>1.5.0-M1</version>
```

Comment:
Is this an actual release or a pre-release that we should avoid?

Reply:
I just chose the latest one in mavenrepository.com. There really isn't much too it. We are only using one method call.